### PR TITLE
Start a dataset description for BIDS compliance

### PR DESCRIPTION
**Pull request by adswa**
Could become a fix to #3 and https://github.com/psychoinformatics-de/studyforrest-data/issues/13

It yet lacks:
- specification of BIDS version (to be determined, and potentially dependent on potential other BIDSification/maintenance efforts)
- DOI (optional, but possible given that a publication to GIN is planned)

### CODE REVIEW

File: datalad_description.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "Name": "studyforrest structural data",
+  "BIDSVersion": "TODO",
+  "DatasetType": "raw",
+  "License": "CC0",
```
Comment (loj) [inline datalad_description.json]: Should be PDDL

**Review by loj — COMMENTED** [review]
Just a small change to the license, so that we're consistent with ourselves.

**Comment by adswa** [comment]
Thank you!

File: datalad_description.json (hunk 1 above)
Comment (adswa) [reply datalad_description.json]:
```suggestion
  "License": "PDDL",
```

**Comment by adswa** [comment]
Should this be merged, or will this be superseded by reconversion efforts? (@loj @bpoldrack )

**Review by mih — APPROVED** [review]
THX

**Pull request merged**